src/style.css

```diff
@@ -0,0 +1,153 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir.ttc");
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: bold;
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+/* Big search bar */
+
+.bigSearchBar {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+
+  font-size: 16px;
+  
+  display: flex;
+  padding-left: 26px;
+  padding-top: 23px;
+  padding-bottom: 21px;
+  margin-top: 20px;
+}
+
+.bigSearchBar:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.bigSearchBar:focus-within {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+
+  filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
+  color: #3d4e61;
+  font-weight: bold;
+}
+
+.bigSearchBarInput {
+  align-self: center;
+  width: 100%;
+  /* height: 66px; */
+  height: 22px;
+  border: none;
+  background-color: transparent;
+  font-weight: bold;
+}
+
+.bigSearchBarInput::placeholder {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  /* line-height: 22px; */
+  color: #3d4e61;
+}
+
+.bigSearchBarInput:focus {
+  outline: none;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+  font-weight: bold;
+  filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
+}
+
+.SearchBarImg {
+  width: 19px;
+  height: 19px;
+  align-self: center;
+  margin-right: 17px;
+}
+
+.bigSearchBarInput:focus + .bigSearchBar {
+  background-color: red;
+}
+
+/* Small search bar */
+
+.smallSearchBar {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+
+  font-size: 16px;
+  line-height: 19px;
+
+  display: flex;
+  padding: 0 15px;
+  padding-left: 13px;
+  margin-top: 20px;
+}
+
+.smallSearchBar:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.smallSearchBarInput {
```

Comment:
you are duplicating a lot of styles
create common `input` class to write shared styles ([checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md), 2nd point)